app/controllers/addresses_controller.rb

```diff
@@ -11,7 +11,9 @@ def create
 
   def show
     findit
```

Comment:
rename this to `find_address` and use a before_action

Reply:
Done and working in the refactor OOD.